tests/post/features/volume.feature

```diff
@@ -269,3 +269,36 @@ Feature: Volume management
         Then the Volume 'test-volume12-lvmlv' is 'Available'
         And the PersistentVolume 'test-volume12-lvmlv' has size '10Gi'
         And the device '/dev/test-vg-12/test-volume12-lvmlv' exists
+
+    Scenario: Test volume re-creation (lvmLogicalVolume force-lvcreate)
```

Comment:
From what I see you forgot to add this scenario in the test file, so this test is not run